hepcrawl/testlib/fixtures.py

```diff
@@ -70,19 +71,25 @@ def get_node(spider, tag, response=None, text=None, rtype="xml"):
     return node
 
 
-def get_responses_path(*path_chunks):
+def get_test_path(test_type, *path_chunks):
```

Comment:
I don't really think the name of this function is good enough :/...

Maybe something like `get_test_suite_path` is better, as it's not really a test you are getting the path to, but the suite of them.